llm-complete-guide/configs/dev/rag_eval.yaml

```diff
@@ -15,3 +15,12 @@ settings:
       - pygithub
       - elasticsearch
     python_package_installer: "uv"
+    
+steps:
+  url_scraper:
+    parameters:
+      docs_url: https://docs.zenml.io/
+      use_dev_set: true
+  index_generator:
+    parameters:
+        index_type: postgres 
```

Comment:
does this really want to be set to postgres, given that pinecone is the default now?

Reply:
+1 on that. let's keep it to pinecone